packages/core/examples/editableTextExample.tsx

```diff
@@ -7,24 +7,27 @@ import * as React from "react";
 
 import { EditableText, Intent, Switch } from "../src";
 import BaseExample, { handleBooleanChange, handleNumberChange } from "./common/baseExample";
-import { IntentSelect } from "./common/intentSelect";;
+import { IntentSelect } from "./common/intentSelect";
 
 export interface IEditableTextExampleState {
     intent?: Intent;
     report?: string;
     selectAllOnFocus?: boolean;
+    shouldConfirmOnEnter?: boolean;
```

Comment:
since the above boolean is `selectAllOnFocus` and not `shouldSelectAllOnFocus` i think your prop should be `confirmOnEnter` instead of `shouldConfirmOnEnter`